packages/component-library/src/components/plmg-tabs/test/plmg-tabs.e2e.ts

```diff
@@ -0,0 +1,39 @@
+import { newE2EPage } from '@stencil/core/testing';
+import { AxePuppeteer } from '@axe-core/puppeteer';
+import { Page } from 'puppeteer';
+
+describe('plmg-tabs', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<plmg-tabs></plmg-tabs>');
+
+    const element = await page.find('plmg-tabs');
+    expect(element).toHaveClass('hydrated');
+  });
+
+  describe('all possible variations', () => {
+    it('are accessible', async () => {
+      const page = await newE2EPage();
+
+      const htmlContent = `
+        <plmg-tabs>
+          <plmg-tab label="Tab 1"></plmg-tab>
+          <plmg-tab icon="home" label="Tab 2"></plmg-tab>
+          <plmg-tab icon="home"></plmg-tab>
```

Comment:
could add `disabled` and `active` in here for fun

Reply:
Addressed in [0a372cf](https://github.com/DuckyTeam/Plumage/pull/61/commits/0a372cf4293603c20b0f1776bfe7bf1dd69c8d81)